src/main/java/frc/robot/RobotContainer.java

```diff
@@ -64,6 +66,10 @@ private void configureButtonBindings() {
     // Instantiate the intake command and bind it
     intakeCommand = new OperatorIntakeCommand();
     new JoystickButton(driverStick, XboxController.Button.kBumperLeft.value).whenHeld(intakeCommand);
+
+    // Instantiate the shooter ramping command and bind it
+    rampShooterCommand = new SpinUpShooter();
+    new JoystickButton(driverStick, XboxController.Button.kA.value).whenPressed(rampShooterCommand);
```

Comment:
How will this work - will it hold the shooter flywheel at speed while the button is pressed down?

Reply:
The command will be scheduled when the button goes from inactive to active. The button doesn't have to be held continuously for the shooter to ramp up. The command can be rebound to another trigger based on drive team preference or anything that makes more sense.